crates/net/network/src/transactions.rs

```diff
@@ -75,6 +74,31 @@ impl TransactionsHandle {
     pub fn propagate(&self, hash: TxHash) {
         self.send(TransactionsCommand::PropagateHash(hash))
     }
+
+    /// Manually propagate the transaction that belongs to the hash to a specific peer.
+    pub fn propagate_hash_to(&self, hash: TxHash, peer: PeerId) {
+        self.send(TransactionsCommand::PropagateHashTo(hash, peer))
+    }
+
+    /// Request the active peer IDs from the [`TransactionsManager`].
+    pub fn get_active_peers(&self) {
+        self.send(TransactionsCommand::GetActivePeers)
+    }
+
+    /// Manually propagate full transactions to a specific peer.
+    pub fn propagate_transactions_to(&self, transactions: Vec<TxHash>, peer: PeerId) {
+        self.send(TransactionsCommand::PropagateTransactionsTo(transactions, peer))
+    }
+
+    /// Request the transaction hashes known by specific peers.
+    pub fn get_transaction_hashes(&self, peers: Vec<PeerId>) {
+        self.send(TransactionsCommand::GetTransactionHashes(peers))
+    }
+
+    /// Request the transaction hashes known by a specific peer.
+    pub fn get_peer_transaction_hashes(&self, peer: PeerId) {
+        self.send(TransactionsCommand::GetPeerTransactionHashes(peer))
+    }
```

Comment:
are these commands that any node can reply with? or just reth nodes?

Reply:
these are just additional commands to interact with the TransactionManager, essentially to interact with the network manually, for sending specific hashes/txs to specific peers manually